static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -130,6 +149,118 @@ function NoContext({isLoading}: NoContextProps) {
   );
 }
 
+type ReleaseContextProps = {
+  api: Client;
+  dataRow: EventData;
+  organization: Organization;
+};
+
+function ReleaseContext(props: ReleaseContextProps) {
+  const {isLoading, isError, data} = useQuery<ReleaseWithHealth>(
+    [`/organizations/${props.organization?.slug}/releases/${props.dataRow.release}/`],
+    undefined,
+    {
+      staleTime: fiveMinutesInMs,
+      retry: false,
+    }
+  );
+
+  const getCommitAuthorTitle = () => {
+    const user = ConfigStore.get('user');
+    const userInAuthors =
+      data &&
+      data.authors.length >= 1 &&
+      data.authors.find((author: User) => author.id && user.id && author.id === user.id);
+    return tct('[commitCount] [commitDesc] by [authorDesc]', {
+      commitCount: data?.commitCount,
+      commitDesc: data?.commitCount !== 1 ? 'commits' : 'commit',
```

Comment:
This comment isn't necessary anymore because we talked about separating this into individual lines, but I think it's useful to know about in the future

Reply:
Ahh yeah will keep in mind